src/server/app.js

```diff
@@ -6,8 +8,20 @@ import http from 'http';
 import logger from 'morgan';
 import session from 'express-session';
 
+import type { $Application, Router } from 'express';
```

Comment:
Why `$Application`?

Reply:
The `express` typing information comes from a community package. That's what they made the name of the type for an Express Application (for some reason). It has to be that for Flow to properly resolve type info.